src/lib/y2network/connection_config/base.rb

```diff
@@ -31,10 +31,12 @@ class Base
       # no specific interface then it could be activated by the first available
       # device.
       #
-      # #FIXME: Maybe it could be a matcher instead of an Interface, or just a
+      # @return [String] Connection name
+      attr_accessor :name
+      # #FIXME: Maybe it could be a matcher instead of an Interface or just
       # the interface name by now.
       #
-      # @return [Interface, nil]
+      # @return [Interface, String, nil]
```

Comment:
document what each type mean. interface is expected. String should be only interface name? and nil is connection without interface?

Reply:
It is just for opening the discussion about what to use by now. In virtual connections, it is mainly a 1-1 relationship between the connection and the virtual interface, for other type of connection it could be a matcher. 

The doubt also arised when working with the readers, who should be responsible of add the virtual interface and what is preferable to use, the name or the interface as the association because at the time of reading the interface is still not present and the reader know nothing about current config..